src/game/Spells/SpellInfo.cpp

```diff
@@ -2013,6 +2013,10 @@ AuraStateType SpellInfo::LoadAuraState() const
     // Faerie Fire (druid versions)
     if (SpellFamilyName == SPELLFAMILY_DRUID && SpellFamilyFlags[0] & 0x400)
         return AURA_STATE_FAERIE_FIRE;
+    
+    // Touch of Zanzil (Rogue quest spell)
+	if (Id == 9991)
+		return AURA_STATE_FAERIE_FIRE;
```

Comment:
also use constants instead of magic numbers, which in this case means: store the ``9991`` number into a constant and then use it like ``if (Id == YOUR_NEW_CONSTANT)``